content/id/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Menambahkan Entry pada /etc/hosts Pod dengan HostAliases
+content_template: templates/concept
+weight: 60
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+Menambahkan _entry_ pada file /etc/hosts Pod akan meng-_override_ 
+resolusi _hostname_ pada level Pod ketika DNS dan opsi lainnya tidak tersedia. 
+Pada versi 1.7, pengguna dapat menambahkan _entry_ kustom beserta _field_ HostAliases 
+pada PodSpec.
+
+Modifikasi yang dilakukan tanpa menggunakan HostAliases tidaklah disarankan 
+karena file ini diatur oleh Kubelet dan dapat di-_override_ ketika Pod dibuat/di-_restart_.
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Isi Default pada File Hosts
+
+Misalnya saja kamu mempunyai sebuah Pod Nginx yang memiliki sebuah IP Pod:
+
+```shell
+kubectl run nginx --image nginx --generator=run-pod/v1
+```
+
+```shell
+pod/nginx created
+```
+
+Perhatikan IP Pod tersebut:
+
+```shell
+kubectl get pods --output=wide
+```
+
+```shell
+NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
+nginx    1/1       Running   0          13s    10.200.0.4   worker0
+```
+
+File hosts yang ada akan tampak sebagai berikut:
+
+```shell
+kubectl exec nginx -- cat /etc/hosts
+```
+
+```none
+# Kubernetes-managed hosts file.
```

Comment:
English comment should be translated, suggestion:

Berkas hosts yang dikelola Kubernetes